sl/content.md

```diff
@@ -0,0 +1,89 @@
+# Scientific Linux
+
+%%LOGO%%
+
+This image containers for [Scientific Linux](http://www.scientificlinux.org/).
+
+The source for each release is in [its own branch](http://github.com/scientificlinux/sl-docker/branches) and contains the kickstart used to build the image.
+
+These images are maintained by the Scientific Linux Team in [github](https://github.com/scientificlinux/sl-docker).
+
+## Container Tags
+
+A tag will be provided for each of the [maintained major releases](http://www.scientificlinux.org/downloads/sl-versions/) of Scientific Linux (`6`, `7`).
+
+The `latest` tag will track the highest version numbered release of Scientific Linux.
+
+Updated images will be released roughly once per month.
+
+## Build Logs
+
+The SL docker containers are built on our internal build system and packaged at [Docker Cloud](https://cloud.docker.com/app/scientificlinux/repository/docker/scientificlinux/sl/builds/).
+
+## Getting Help
+
+* [SL Faq](https://www.scientificlinux.org/documentation/faq)
+
+* [Email Lists](https://www.scientificlinux.org/community)
+
+## Docker, overlayfs, and yum
+
+Recent Docker versions support the [overlayfs](https://docs.docker.com/engine/userguide/storagedriver/overlayfs-driver/) backend, which is enabled by default on most distros supporting it from Docker 1.13 onwards. On SL 6 and 7, that backend requires `yum-plugin-ovl` to be installed and enabled, which it is in our containers. Make it sure you retain the `plugins=1` option in `/etc/yum.conf` if you update that file; otherwise, you may encounter errors related to rpmdb checksum failure - see [Docker ticket 10180](https://github.com/docker/docker/issues/10180) for more details.
+
+## Package docs and licence files
+
+By default the SL docker images do not include these files.  If you require them, please remove `tsflags=nodocs` from `/etc/yum.conf` and run `yum reinstall mypackage` to recieve the documentation.
+
+---
+# Quick Reference
+
+## Example Usage
+You can try out the containers via:
+
+```
+$ docker pull scientificlinux/sl
+$ docker run -it %%IMAGE%%:6 cat /etc/redhat-release
+$ docker run -it %%IMAGE%%:7 cat /etc/redhat-release
+```
+
+## Enabling systemd in SL7
+The SL7 docker container ships with systemd mostly functional.  You can build a SL7 systemd enabled container with the following Dockerfile
+
+In order to run a container with systemd, you will need to mount the cgroups volumes from the host.
+
+```
+# Example SL7 systemd Dockerfile
+FROM %%IMAGE%%:7
+ENV container docker
+### To enable apache within the container uncomment the next two lines
+# RUN yum -y install httpd; yum clean all; systemctl enable httpd.service
```

Comment:
Shouldn't this be using `&&` instead of `;` so that it short-circuits on failure? (or `set -e;` / `set -ex;` at the beginning)

Reply:
Why not just leave it uncommented and have an initial comment like `# Example enabling httpd in systemd`?